src/components/ui/modal/MyModal.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import classes from './MyModal.module.css'
+const MyModal = (props) => {
+    const modalClasses = [classes.myModal]
```

Comment:
2. You can use template strings to form classes:
```suggestion
    const modalClasses = [myModal, visible && active];
```